diff --git a/webextension/scripts/utils.js b/webextension/scripts/utils.js
index faa711f..24f5b17 100644
--- a/webextension/scripts/utils.js
+++ b/webextension/scripts/utils.js
@@ -840,7 +840,7 @@ function initDefaultOptions () {
 function afterAcceptTerms () {
   chrome.storage.local.set({
     agreement: true,
-    private_mode_setting: false,
+    private_mode_setting: true,
     not_found_setting: true
   })
   chrome.action.setPopup({ popup: chrome.runtime.getURL('index.html') }, checkLastError)
